lib/benchpark/cmd/analyze.py

```diff
@@ -300,6 +334,15 @@ def prepare_data(**kwargs):
 
     # Remove singular roots if inclusive metric
     metric = kwargs["yaxis_metric"]
+
+    tk.dataframe["Bandwidth (GB/s)"] = (
+        tk.dataframe["Bytes/Rep"]
+        / tk.dataframe["Avg time/rank (exc)"]
+        / 10**9
+        * tk.dataframe["Reps"]
+        * tk.metadata["mpi.world.size"]
+    )
```

Comment:
This formula only works for RAJAPerf data

Reply:
resolved by [24754d8](https://github.com/LLNL/benchpark/pull/1132/commits/24754d8ff93c193b7feed12c517514d33ba68838)